nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# {{ cookiecutter.pipeline_slug }} Contributing Guidelines
```

Comment:
`{{ cookiecutter.pipeline_slug }}` is just the final name of the pipeline (eg. `rnaseq`). So please do a find and replace to change to `nf-core/{{ cookiecutter.pipeline_slug }}` everywhere to include that base.